src/doc/docs/decomposition.md

```diff
@@ -3,60 +3,58 @@
     Guide for writing re-usable bundles
 
-In dropwizard we have only one decomposition element: [ConfiguredBundle](https://www.dropwizard.io/en/stable/manual/core.html#bundles).
+In dropwizard there is only one decomposition element: [ConfiguredBundle](https://www.dropwizard.io/en/stable/manual/core.html#bundles).
 
 !!! note
-    In pure dropwizard, bundles also used inside single application to separate configuration blocks
-    (simplify logic). In guicey, this is not required as [classpath scan](guide/scan.md) could be
-    used for extensions registration (configuration amount reduced).
+    In pure dropwizard, bundles may also  be used within a single application to separate configuration blocks
+    (simplify logic). In guicey, this is not required as [classpath scan](guide/scan.md) may be
+    used for extension registration and reduce the amount of required configuration.
     
-    This chapter describe only re-usable logc decomposition.
+    This chapter describes only re-usable logic decomposition.
     
-In guicey we have: guicey bundle (`GuiceyBundle`), guice module (`Module`) and dropwizard bundle (`ConfiguredBundle`).
-This could confuse.
+In guicey there are three decomposition elements - guicey bundles (`GuiceyBundle`), guice modules (`Module`) and dropwizard bundle (`ConfiguredBundle`).
+Having three options could be confusing.
 
 - There are existing [dropwizard modules](https://modules.dropwizard.io/thirdparty/) - `ConfiguredBundle`
-- Existing [guice modules](https://github.com/google/guice/wiki/3rdPartyModules) (outdated list, just as an exampel) - `Module`
+- Existing [guice modules](https://github.com/google/guice/wiki/3rdPartyModules) (outdated list, just as an example) - `Module`
 - And [guicey extensions](guide/modules.md) - `GuiceyBundle`
 
-All these modules are supposed to be used together (obviously). In some cases, 
-guicey explicitly provides wrapping modules (e.g. [jdbi](extras/jdbi3.md) - wrapper for dropwizard module),
-but such wrappers usually provide additional abilities (impossible with pure module) and
-not driven by inability to use a raw module.
+All of these modules are supposed to be used together. In some cases, guicey explicitly provides wrapping 
+modules (e.g. the [jdbi](extras/jdbi3.md) wrapper around the [dropwizard module](https://www.dropwizard.io/en/latest/manual/jdbi3.html)). 
+Such wrappers usually provide additional features and enhancements that would be impossible with a single module type.
 
 ## Guicey bundle
 
-As it [was described](concepts.md#bundles), guicey have to introduce its own [bundle](guide/bundles.md) because guicey provides
-additional configuration features (even in the simplest case you should be able to
-configure guice modules).
+As it [was described](concepts.md#bundles), guicey introduces its own [bundle](guide/bundles.md) because guicey provides
+additional configuration features. Even the simple use cases require some configuration of guice modules.
```

Comment:
"Even the simple use cases require some configuration of guice modules." -> No, it's not strictly required, but may be requirwed - the simplest example is a need of gucie module registration from bundle. Maybe it should be like: "In the simplest case, you may need to apply guice modules from bundle"